### PR TITLE
EmptySet with S

### LINKED ISSUE #16337: EmptySet and UniversalSet without S
I had thought we cleaned up all the S objects, but I guess not. 

Right now we have

```py
>>> from sympy import *
>>> EmptySet
<class 'sympy.sets.sets.EmptySet'>
>>> UniversalSet
UniversalSet()
```

There are two issues:

- `EmptySet` should be the instance, not the class.

- UniversalSet should not print with `()`.

### PR DESCRIPTION
**Pull request by dnabanita7**
Since,Singleton classes have instances that are the same name as the class.So,
EmptySet should return its instance rather than its class name.
As well,I changed the test_args.py to test ``if S.EmptySet is EmptySet()``.
Fixes #16337

* sets
     * changed __init__.py to show instance of EmptySet when being called rather than showing its classname

### CODE REVIEW

File: sympy/concrete/summations.py [hunk 1]
```diff
@@ -946,9 +946,15 @@ def eval_sum(f, limits):
         return eval_sum_direct(f, (i, a, b))
     if isinstance(f, Piecewise):
         return None
+    #If the number of terms is infinite and series is convergent
+    #if not(f.is_convergent) and expr.subs(1,a+i).is_positive:
+    #    return oo
+    # elif not(f.is_convergent) and expr.subs(i,a+i).is_negative:
+    #    return -oo
```
Comment (oscarbenjamin) [inline sympy/concrete/summations.py]: Are these lines left over from debugging?

File: sympy/concrete/summations.py [hunk 2]
```diff
@@ -946,9 +946,15 @@ def eval_sum(f, limits):
...
     # Try to do it symbolically. Even when the number of terms is known,
     # this can save time when b-a is big.
     # We should try to transform to partial fractions
+    #if b != oo or a == -oo:
```
Comment (oscarbenjamin) [inline sympy/concrete/summations.py]: And this one?

**Comment by oscarbenjamin** [comment]
I'm not sure I understand what the changes here are doing. Can you give clear examples of input and output before and after this change?

File: sympy/concrete/summations.py (hunk 1 above)
Comment (Naba7) [reply sympy/concrete/summations.py]: Yes,but this is from other PR.Then,I didn't know how to create branches.So,here is the problem.I will clear this thing over here

File: sympy/concrete/summations.py (hunk 2 above)
Comment (Naba7) [reply sympy/concrete/summations.py]: same mistake

**Comment by Naba7** [comment]
I have just changed the __init__.py .Plus I am changing importing of EmptySet in different functions.I am making a pr  for that.

**Comment by Naba7** [comment]
Although,I have changed the import error of EmptySet,I still get the error that EmptySet can't be
dispatched as it is a non-type object as it is getting its signature from S.EmptySet not from
sympy.sets.sets

**Comment by asmeurer** [comment]
Travis might not be running because of the merge conflicts. I would suggest fixing them (you will need to do it anyway).

**Pull request closed without merging**